packages/cli/src/commands/locations/rooms/update.ts

```diff
@@ -0,0 +1,49 @@
+import { flags } from '@oclif/command'
+import { Room, RoomRequest } from '@smartthings/core-sdk'
+
+import { buildTableOutput, getRoomsByLocation } from '../rooms'
+import { SelectingInputOutputAPICommand } from '@smartthings/cli-lib'
+
+
+export default class RoomsUpdateCommand extends SelectingInputOutputAPICommand <RoomRequest, Room, Room> {
+	static description = 'update a room'
+
+	static flags = {
+		...SelectingInputOutputAPICommand.flags,
+		locationId: flags.string({
+			char: 'l',
+			description: 'a specific locationId to query',
+		}),
+	}
+
+	static args = [{
+		name: 'idOrIndex',
```

Comment:
This doesn't affect how it works but technically, this is just an `id` for the `Selecting...` classes. (They can't specify an index on the command line--only when prompted.)